src/index.js

```diff
@@ -1,47 +1,54 @@
-import colors from './styles/colors/index'
-import Button from './components/Buttons/Button'
-import CircleButton from './components/Buttons/CircleButton'
-import Column from './components/Grid/Column'
-import GlobalTheme from './styles/GlobalTheme'
-import Grid from './components/Grid/Grid'
-import Icon from './components/Icon/Icon'
-import Link from './components/Link/Link'
-import NavigationPills from './components/NavigationPills/NavigationPills'
-import Normalize from './styles/Normalize'
-import responsive from './styles/responsive'
-import Row from './components/Grid/Row'
-import ScrollTrack from './components/ScrollTrack/ScrollTrack'
-import SetStyles from './styles/SetStyles'
-import spacing from './styles/spacing'
-import TextField from './components/Forms/TextField'
-import ServerError from './components/Forms/ServerError'
-import ValidationError from './components/Forms/ValidationError'
-import FloatingLabel from './components/Forms/FloatingLabel'
-import zIndex from './styles/zIndex'
-import themer from './styles/themer/index'
-import withTheme from './styles/themer/withTheme'
+import colors           from './styles/colors/index'
+import Button           from './components/Buttons/Button'
+import CircleButton     from './components/Buttons/CircleButton'
+import RadioCheckbox    from './components/Buttons/RadioCheckbox'
+import Column           from './components/Grid/Column'
+import GlobalTheme      from './styles/GlobalTheme'
+import Grid             from './components/Grid/Grid'
+import Icon             from './components/Icon/Icon'
+import Link             from './components/Link/Link'
+import NavigationPills  from './components/NavigationPills/NavigationPills'
+import Normalize        from './styles/Normalize'
+import responsive       from './styles/responsive'
+import Row              from './components/Grid/Row'
+import ScrollTrack      from './components/ScrollTrack/ScrollTrack'
+import SetStyles        from './styles/SetStyles'
+import spacing          from './styles/spacing'
+import TextField        from './components/Forms/TextField'
+import ServerError      from './components/Forms/ServerError'
+import ValidationError  from './components/Forms/ValidationError'
+import FloatingLabel    from './components/Forms/FloatingLabel'
+import zIndex           from './styles/zIndex'
+import themer           from './styles/themer/index'
+import withTheme        from './styles/themer/withTheme'
```

Comment:
I'd prefer we don't align imports to be consistent with the rest of the lib, but open to other arguments